packages/zoe/src/zoeService/startInstance.js

```diff
@@ -143,7 +143,7 @@ export const makeStartInstance = (
   );
 
   const vivifyEmptyFacet = facetName =>
-    vivifyFarClass(
+    defineExoFactory(
```

Comment:
We still have vivify in several places. In particular, a contract is a file that exports a `start` or `vivify` function.  But there are several other places in the implementation of durable types that have functions called vivify. Should we also try to get rid of them, since the rationale for "vivify" won't be ubiquitous?

Reply:
Good suggestions. Take a look at the second commit, where I 
   * replace (`*Class`,`*Factory`) with `*Maker`
   * replace (`vivify*`,`define*`) with `prepare*`

Does anyone prefer the first (`*Factory`,`define*`) over the second (`*Maker`,`prepare*`)? Is anyone not ok with the second?

Reply:
Those both seem like improvements to me.